vncviewer/OptionsDialog.cxx

```diff
@@ -63,7 +64,7 @@ std::map<OptionsCallback*, void*> OptionsDialog::callbacks;
 static std::set<OptionsDialog *> instances;
 
 OptionsDialog::OptionsDialog()
-  : Fl_Window(580, 480, _("TigerVNC Options"))
+  : Fl_Window(580, 420, _("TigerVNC Options"))
```

Comment:
Please include this commit into the main one so we can avoid this back and forth.